### PR TITLE
Service UI Components

### PR DESCRIPTION
**Pull request by AbdulrahmanCad**

### CODE REVIEW

File: src/components/modal/ServiceModal.js [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+  height: 40, 
+  backgroundColor: "#F4F3EE",
+  width: "100%",
+};
+
+export default function BasicModal({ serviceModal, setServiceModal }) {
+  const [showPassword, setShowPassword] = React.useState(false);
+  const [error, setError] = React.useState("");
+  const [showPassword2, setShowPassword2] = React.useState(false);
+  const [showPassword3, setShowPassword3] = React.useState(false);
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: can we rename these variable with more descriptive one ?

File: src/components/modal/ServiceModal.js [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+                    display: "flex",
+                    justifyContent: "end",
+                    color: "#A3A4A2",
+                    minWidth: "2rem",
+                    minHeight: "2.5rem",
+                    animationTimeline: 5000,
+                    "&:hover": {
+                      opacity: 1,
+                      boxShadow: 4,
+                    },
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: can remove inline styles and replace them with classes ?

File: src/components/modal/ServiceModal.js [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+                        onChange={onChange}
+                        placeholder="Enter name"
+                      />
+                    </Grid>
+                  </Typography>
+                </FormControl>
+
+                <FormControl
+                  variant="outlined"
+                  sx={{ my: 1.5, color: "#112849" }}
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+                        onChange={onChange}
+                        placeholder="Enter description"
+                      />
+                    </Grid>
+                  </Typography>
+                </FormControl>
+
+                <FormControl
+                  variant="outlined"
+                  sx={{ my: 1.5, color: "#112849" }}
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+                        onChange={onChange}
+                        placeholder="Enter path"
+                      />
+                    </Grid>
+                  </Typography>
+                </FormControl>
+
+                <FormControl
+                  variant="outlined"
+                  sx={{ my: 1.5, color: "#112849" }}
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+                  </Typography>
+                </FormControl>
+
+                <Typography sx={{ color: "red" }}>{error}</Typography>
+                <Box
+                  height="2.5rem"
+                  mt="0.5rem"
+                  display="flex"
+                  justifyContent="end"
+                >
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js:382]: inline styles

File: src/components/modal/ServiceModal.js [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+                      color: "#A3A4A2",
+                      my: 1.5,
+                      mr: 1,
+                      fontSize: 16,
+                      animationTimeline: 5000,
+                      "&:hover": {
+                        opacity: 1,
+                        color: "#FC574E",
+                      },
+                    }}
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+                      padding: "6px 12px",
+                      borderRadius: "6px",
+                      backgroundColor: "#FC574E",
+                      color: "white",
+                      "&:hover": {
+                        backgroundColor: "#CA463E",
+                        boxShadow: "none",
+                      },
+                    }}
+                  >
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 9]
```diff
@@ -0,0 +1,490 @@
...
+          >
+            <Box sx={SecondModalStyle}>
+              <Box display="flex" justifyContent="center">
+                <Avatar
+                  sx={{
+                    alignItems: "center",
+                    bgcolor: "#e8f3eb",
+                    width: 70,
+                    height: 70,
+                  }}
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/components/modal/ServiceModal.js [hunk 10]
```diff
@@ -0,0 +1,490 @@
...
+                  fontSize: 16,
+                  borderRadius: "6px",
+                  backgroundColor: "#ebe9e1",
+                  color: "#363d43",
+                  "&:hover": {
+                    backgroundColor: "#C7C7C1",
+                  },
+                }}
+              >
+                Ok
```
Comment (Abdullah-Sanji) [inline src/components/modal/ServiceModal.js]: inline styles

File: src/pages/Endpoint.js [hunk 11]
```diff
@@ -232,12 +236,61 @@ export default function Endpoint() {
...
+                  sx={{
+                    ":hover": {
+                      bgcolor: "#ebe9e1",
+                      color: "#7e8282",
+                    },
+                    border: 1,
+                    borderColor: "#EBE9E1",
+                    borderRadius: 2,
+                    marginRight: "1rem"
+                  }}
```
Comment (Abdullah-Sanji) [inline src/pages/Endpoint.js:253]: inline styles

File: src/pages/Endpoint.js [hunk 12]
```diff
@@ -232,12 +236,61 @@ export default function Endpoint() {
...
+                    ":hover": {
+                      bgcolor: "#CA463E",
+                      color: "white",
+                    },
+                    color: "white",
+                    textTransform: 'none',
+                    backgroundColor: "#FC574E",
+                  }}
+                  variant="contained"
+                >
```
Comment (Abdullah-Sanji) [inline src/pages/Endpoint.js:280]: inline styles

File: src/pages/Service.js [hunk 13]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Service() {
+  return (
+    <div>Service</div>
+  )
+}
+
+export default Service
```
Comment (Abdullah-Sanji) [inline src/pages/Service.js]: why do we have empty service here ?

File: src/components/modal/ServiceModal.js (hunk 2 above)
Comment (AbdulrahmanCad) [reply src/components/modal/ServiceModal.js]: fixed

**Review by Abdullah-Sanji — APPROVED** [review]

**Pull request merged**